kubetest/gke.go

```diff
@@ -102,6 +102,8 @@ type gkeDeployer struct {
 	setup          bool
 	kubecfg        string
 	instanceGroups []*ig
+	bastionIPName  string
+	zoneRegion     string
```

Comment:
What is zoneRegion?

Reply:
changed to regionCache - used in getRegion()

Reply:
removed after refactoring